dcos_installer/test_backend.py

```diff
@@ -67,6 +67,35 @@ def test_set_superuser_password(tmpdir):
         assert passlib.hash.sha512_crypt.verify('foo', config['superuser_password_hash'])
 
 
+def test_generate_node_upgrade_script(tmpdir):
+    upgrade_config = """
+---
+# The name of your DC/OS cluster. Visable in the DC/OS user interface.
+cluster_name: 'DC/OS'
+master_discovery: static
+exhibitor_storage_backend: 'static'
+resolvers:
+- 8.8.8.8
+- 8.8.4.4
+ssh_port: 22
+process_timeout: 10000
+bootstrap_url: file:///opt/dcos_install_tmp
+master_list: ['10.0.0.1', '10.0.0.2', '10.0.0.5']
+"""
+
+    with tmpdir.as_cwd():
+        create_fake_build_artifacts(upgrade_config, tmpdir)
+
+    output = subprocess.check_output(['dcos_installer', '--generate-node-upgrade-script', 'fake'], cwd=str(tmpdir))
+    assert output.decode('utf-8').splitlines()[-1].split("Node upgrade script URL: ", 1)[1]\
+                                                  .endswith("dcos_node_upgrade.sh")
```

Comment:
It is split by line, I am using splitlines and look at the last line. As for the name of the script i want it in here incase we mess up with a typo or something for unit test.